src/pages/strategies/portfolio/index.tsx

```diff
@@ -1,26 +1,23 @@
 import { Row } from '@tanstack/react-table';
 import { PortfolioAllTokens } from 'components/strategies/portfolio';
+import { GetPortfolioTokenHref } from 'components/strategies/portfolio/types';
 import { PortfolioData } from 'components/strategies/portfolio/usePortfolioData';
 import { useStrategyCtx } from 'hooks/useStrategies';
-import { Pathnames, PathParams, useNavigate } from 'libs/routing';
-
-export type GetPortfolioTokenHref = (row: PortfolioData) => {
-  href: Pathnames;
-  params: PathParams;
-};
-
-export const getPortfolioTokenHref: GetPortfolioTokenHref = (row) => ({
-  href: '/strategies/portfolio/token/$address',
-  params: { address: row.token.address },
-});
+import { useNavigate } from 'libs/routing';
 
 export const StrategiesPortfolioPage = () => {
   const { strategies, isLoading } = useStrategyCtx();
   const navigate = useNavigate();
+  const href = '/strategies/portfolio/token/$address';
+
+  const getPortfolioTokenHref: GetPortfolioTokenHref = (row) => ({
+    href,
+    params: { address: row.token.address },
+  });
 
   const onRowClick = (row: Row<PortfolioData>) =>
     navigate({
-      to: '/strategies/portfolio/token/$address',
+      to: href,
```

Comment:
I think you're loosing some typing here. Check with @pingustar what the best strategy here with the router.

Reply:
Changing `href` throws a type error

![image](https://github.com/bancorprotocol/carbon-app/assets/66181828/5c11525e-9c1e-4610-80cc-4c48f32e246b)